plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/BaseCassandraConnectorTest.java

```diff
@@ -553,6 +686,7 @@ public void testNestedCollectionType()
     }
 
     @Test
+    @Override
     public void testInsert()
```

Comment:
In case of `testDelete`  we'll have to stick with the overridden method.

```
io.trino.spi.TrinoException: Deleting without partition key is not supported
```

Thanks for the tip regarding `testInsert` . I've renamed the method to `testAllTypesInsert`